doc/man3/SSL_get_error.pod

```diff
@@ -25,6 +25,15 @@ other OpenSSL function calls should appear in between.  The current
 thread's error queue must be empty before the TLS/SSL I/O operation is
 attempted, or SSL_get_error() will not work reliably.
 
+=head1 NOTES
+
+Some TLS implementations do not send a close_notify alert on shutdown. In such
+cases and in other cases of unexpected EOF, before OpenSSL 3.0,
+B<SSL_ERROR_SYSCALL> with errno=0 was reported.
+
+Since OpenSSL 3.0 the returned error is B<SSL_ERROR_SSL> with a meaningful
+error on the error stack. See the B<SSL_ERROR_ZERO_RETURN> below.
```

Comment:
I suggest:
On an unexpected EOF, versions before OpenSSL 3.0 returned B<SSL_ERROR_SYSCALL>, nothing was added to the error stack, and errno was 0. Since OpenSSL 3.0 the returned error is B<SSL_ERROR_SSL> with a meaningful
error on the error stack.

Reply:
Applied. Thanks!